### PR TITLE
[QRadar v3] Metrics

### PR DESCRIPTION
**Pull request by shmuel44**
## Related Issues

fixes: [link](https://jira-dc.paloaltonetworks.com/browse/CIAC-10619) to the issue
related: [link](https://jira-dc.paloaltonetworks.com/browse/CIAC-10628) 

## Description

Added support for API Execution Metric reporting for QRadar commands, excluding long-running commands such as fetch incidents.

### CODE REVIEW

**Comment by ShirleyDenkberg** [comment]
@jbabazadeh Doc review completed.

File: Packs/QRadar/Integrations/QRadar_v3/README.md [hunk 1]
```diff
@@ -129,6 +129,14 @@ Alternatively, the [retrieve events command](#qradar-search-retrieve-events) can
 If the command takes too long to finish executing, try setting the `interval_in_seconds` to a lower value (down to a minimum of 10 seconds).
 
 
+### API Call Metrics
+
+This feature provides support for tracking the  QRadar API calls.
```
Comment (shmuel44) [inline Packs/QRadar/Integrations/QRadar_v3/README.md:134]:
```suggestion
This feature provides support for tracking the QRadar API calls.
```

File: Packs/QRadar/Integrations/QRadar_v3/README.md [hunk 2]
```diff
@@ -129,6 +129,14 @@ Alternatively, the [retrieve events command](#qradar-search-retrieve-events) can
 If the command takes too long to finish executing, try setting the `interval_in_seconds` to a lower value (down to a minimum of 10 seconds).
 
 
+### API Call Metrics
+
+This feature provides support for tracking the QRadar API calls.
+
+Please note that this feature does not include metrics for long-running commands such as `fetch incidents`.
+
+To view the API Call Metrics, navigate to the **Dashboard & Reports** section in the side-bar and than to the **API Execution Metrics** dashboard.
```
Comment (dorschw) [inline Packs/QRadar/Integrations/QRadar_v3/README.md:138]:
```suggestion
API Metrics are shown in the built-in **API Execution Metrics** dashboard, and are available to use in custom widgets.
```
Please run it by a tech writer

File: Packs/QRadar/Integrations/QRadar_v3/README.md [hunk 3]
```diff
@@ -129,6 +129,14 @@ Alternatively, the [retrieve events command](#qradar-search-retrieve-events) can
 If the command takes too long to finish executing, try setting the `interval_in_seconds` to a lower value (down to a minimum of 10 seconds).
 
 
+### API Call Metrics
+
+This feature provides support for tracking the QRadar API calls.
+
+Please note that this feature does not include metrics for long-running commands such as `fetch incidents`.
```
Comment (dorschw) [inline Packs/QRadar/Integrations/QRadar_v3/README.md:136]:
```suggestion
API Call metrics are not available for long-running commands such as `fetch incidents`.
```

File: Packs/QRadar/Integrations/QRadar_v3/README.md [hunk 4]
```diff
@@ -129,6 +129,14 @@ Alternatively, the [retrieve events command](#qradar-search-retrieve-events) can
 If the command takes too long to finish executing, try setting the `interval_in_seconds` to a lower value (down to a minimum of 10 seconds).
 
 
+### API Call Metrics
+
+This feature provides support for tracking the QRadar API calls.
```
Comment (dorschw) [inline Packs/QRadar/Integrations/QRadar_v3/README.md:134]:
```suggestion
This feature collects metadata on QRadar API calls and their success status.
```

**Review by dorschw — APPROVED** [review]

File: Packs/QRadar/ReleaseNotes/2_5_0.md [hunk 5]
```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+
+##### IBM QRadar v3
+
+Added support for API Execution Metric reporting for QRadar commands, excluding long-running commands such as fetch incidents.
```
Comment (ShirleyDenkberg) [inline Packs/QRadar/ReleaseNotes/2_5_0.md:6]: Should this be API Call metric

**Comment by ShirleyDenkberg** [comment]
@shmuel44 Doc re-review completed.

File: Packs/QRadar/Integrations/QRadar_v3/README.md (hunk 2 above)
Comment (ShirleyDenkberg) [reply Packs/QRadar/Integrations/QRadar_v3/README.md]: From tech writer: Looks good.

**Pull request merged**